examples/allocation/tinygo/testdata/greet.go

```diff
@@ -53,7 +56,7 @@ func _greet(ptr, size uint32) {
 func _greeting(ptr, size uint32) (ptrSize uint64) {
 	name := ptrToString(ptr, size)
 	g := greeting(name)
-	ptr, size = stringToPtr(g)
+	ptr, size = stringToNativePtr(g)
```

Comment:
so I mistook this as a rename, when it is a second function. Go doesn't have a way to name this, I'm aware of, but I think the main thing is to not say "native".

Maybe `stringToLeakedPtr` instead, as the essential thing here is that the remote caller is taking ownership of the pointer. At least our rust example calls this leak, so it is somewhat consistent.

Reply:
done